sql/transaction-isolation.md

```diff
@@ -0,0 +1,73 @@
+---
+title: TIDB 事务隔离级别
+category: user guide
+---
+
+
+# TIDB 事务隔离级别
+
+事务隔离级别是数据库事务处理的基础，ACID 中 I，即 isolation，指的就是事务的隔离性。
+
+sql 92标准定义了4种隔离级别，读未提交，读已提交，可重复读，串行化，见下表。
+
+| isolation Level  | Dirty Read   | Nonrepeatable Read | Phantom Read          | Serialization Anomaly |
+| ---------------- | ------------ | ------------------ | --------------------- | --------------------- |
+| Read uncommitted | Possible     | Possible           | Possible              | Possible              |
+| Read committed   | Not possible | Possible           | Possible              | Possible              |
+| Repeatable read  | Not possible | Not possible       | Not possible in  TIDB | Possible              |
+| Serializable     | Not possible | Not possible       | Not possible          | Not possible          |
+
+TIDB实现了其中的两种，读已提交和串行化。
```

Comment:
1. TIDB -> TiDB
2. Leave a whitespace between English and Chinese characters.
3. Please fix all TIDB -> TiDB, and add whitespace between English and Chinese characters in this PR.

Reply:
ok